STACpopulator/stac_utils.py

```diff
@@ -190,6 +190,9 @@ class MySTACItem(Item):
         )
 
         item = pystac.Item(**json.loads(item.model_dump_json(by_alias=True)))
+        link = pystac.Link(rel="source", target=attrs["catalog_url"], media_type="text/html", title="thredds:" + attrs[
+            "id"])
```

Comment:
Although I don't expect to have another `thredds` service on our instances, this `thredds:` value should be infered from the Magpie service-name before `/catalog/` part.